mz_bokeh_package/utilities/environment.py

```diff
@@ -37,15 +37,18 @@ def get_request_url(cls, endpoint: str) -> str:
         """
 
         env = cls.get_environment()
+        add_an_s = ""
 
         if env == 'staging':
             host = 'staging.materials.zone:5000'
         elif env == 'production':
             host = 'production.materials.zone:5000'
         elif env == 'dev':
             host = os.getenv('API_HOST', 'staging.materials.zone:5000')
+            if host.find('api') > -1:
+                add_an_s = "s"
 
-        return f"http://{host}/{endpoint}"
+        return f"http{add_an_s}://{host}/{endpoint}"
```

Comment:
@captKuk the `add_an_s` variable is not necessary (and a bit "hacky"). I would just change this function to the following, and by this shifting the API traffic to the Kubernetes deployment:

```
env = cls.get_environment()
        add_an_s = ""

        if env == 'staging':
            host = 'https://api-staging.materials.zone/v1beta1'
        elif env == 'production':
            host = 'https://api.materials.zone/v1beta1'
        elif env == 'dev':
            host = os.getenv('API_HOST', 'https://api-staging.materials.zone/v1beta1')

        return f"http://{host}/{endpoint}"
```

Developers that are using the API_HOST environment variable will need to adapt their environment variable accordingly.